r-intro.Rmd

```diff
@@ -24,7 +24,7 @@ RStudio has a large number of useful keyboard shortcuts. A list of these can be
 
 The RStudio team has developed [a number of "cheatsheets"](https://www.rstudio.com/resources/cheatsheets/){target="_blank"} for working with both `R` and RStudio. [This particular cheatsheet for "Base" `R`](https://github.com/rstudio/cheatsheets/blob/main/base-r.pdf){target="_blank"} will summarize many of the concepts in this document. ("Base" `R` is a name used to differentiate the practice of using built-in `R` functions, as opposed to using functions from outside packages, in particular, those from the [`tidyverse`](https://www.tidyverse.org/){target="_blank"}. More on this later.)
 
-When programming, it is often a good practice to follow a style guide. (Where do spaces go? Tabs or spaces? Underscores or CamelCase when naming variables?) No style guide is "correct" but it helps to be aware of what others do. The more important thing is to be consistent within your own code.
+When programming, it is often a good practice to follow a style guide. (Where do spaces go? Tabs or spaces? Underscores or CamelCase when naming variables?) No style guide is "correct," but it helps to be aware of what others do. The more important thing is to be consistent within your own code.
```

Comment:
I like the other changes in this commit, but I would omit this one.

Reply:
@djoldman 
Thank you for your review. I think we should have a comma before "but" when it connects two independent clauses. Any feedback would be appreciated.